platform/mock-local-home-platform.ts

```diff
@@ -81,6 +82,14 @@ export class MockLocalHomePlatform implements MockUDPListener {
     });
   }
 
+  public getIsHomeAppReady(): boolean {
```

Comment:
Does this need to be a `get`? As it's a boolean, having `isHomeAppReady` would imply a getter for a boolean.

Reply:
I changed the boolean name and the getter to `homeAppReady` and `isHomeAppReady`, respectively.  Let me know if this isn't what you meant.